schema.json

```
@@ -459,77 +490,99 @@
             ]
         },
         "orcid": {
+            "description": "Identifier for an author, see https://orcid.org.",
             "format": "uri",
             "pattern": "https://orcid\\.org/[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]{1}",
             "type": "string"
         },
         "person": {
+            "description": "A person.",
             "type": "object",
             "additionalProperties": false,
             "properties": {
                 "address": {
-                    "$ref": "#/definitions/address"
+                    "$ref": "#/definitions/address",
+                    "description": "The person's address."
                 },
                 "affiliation": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "The person's affilitation."
                 },
                 "alias": {
-                    "$ref": "#/definitions/alias"
+                    "$ref": "#/definitions/alias",
+                    "description": "The person's alias."
                 },
                 "city": {
-                    "$ref": "#/definitions/city"
+                    "$ref": "#/definitions/city",
+                    "description": "The person's city."
                 },
                 "country": {
-                    "$ref": "#/definitions/country"
+                    "$ref": "#/definitions/country",
+                    "description": "The person's country."
                 },
                 "email": {
-                    "$ref": "#/definitions/email"
+                    "$ref": "#/definitions/email",
+                    "description": "The person's email address."
                 },
                 "family-names": {
+                    "description": "The person's family names.",
                     "type": "string"
                 },
                 "fax": {
-                    "$ref": "#/definitions/fax"
+                    "$ref": "#/definitions/fax",
+                    "description": "The person's fax number."
                 },
                 "given-names": {
+                    "description": "The person's given names.",
                     "type": "string"
                 },
                 "name-particle": {
+                    "description": "The person's name-particle.",
                     "type": "string"
                 },
                 "name-suffix": {
+                    "description": "The person's name-suffix.",
```

```suggestion
                    "description": "The person's name-suffix.",
                    "examples": [
                        "'Jr.' (for Sammy Davis Jr.)",
                        "'III' (for Frank Edwin Wright III)"
                    ],
```

Add examples for name suffixes.